release.sh

```diff
@@ -1,24 +1,25 @@
 #!/usr/bin/env bash
 
-set -e
+set -eu
 
-[ $# == 1 ] || { echo "usage: $0 version" && exit 1; }
+if [[ $# != 1 ]]; then
+	echo "usage: $0 vX.Y.Z"
+	exit 1
+fi
 
 VER=$1
 
-[[ "$VER" =~ ^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+$ ]] || {
+if ! [[ "$VER" =~ ^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+$ ]]; then
 	echo "malformed version: \"$VER\""
-	exit 2
-}
+	exit 1
+fi
 
-if test -n "$(git ls-files --others | \
-	grep --invert-match '\(vendor\|build/operator-sdk-v.\+\)')";
-then
+if git ls-files --others | grep -Ev 'build/operator-sdk-v.+'; then
```

Comment:
should not it ignore the website ? 

$ git ls-files --others | grep -Ev 'build/operator-sdk-v.+'
test/test-framework/build/_output/bin/test-framework
test/test-framework/go.sum
website/resources/_gen/assets/scss/scss/main.scss_9fadf33d895a46083cdd64396b57ef68.content
website/resources/_gen/assets/scss/scss/main.scss_9fadf33d895a46083cdd64396b57ef68.json